deploy/kubernetes/scripts/generate-certs.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+[ -z `which make` ] && echo "You need to install make first." && exit 1
```

Comment:
deploy-k8s.sh should add 
if [ $? != 0 ] after generate-certs.sh

Deploy failed should be print.